plugins/discourse-rewind/plugin.rb

```diff
@@ -27,6 +27,34 @@ module ::DiscourseRewind
   def self.public_asset_path(name)
     File.expand_path(File.join(__dir__, "public", name))
   end
+
+  def self.year_date_range(date_override = nil)
```

Comment:
Mostly for easy debugging purposes, you can do `DiscourseRewind.year_date_range`